proto/storage/policy.proto

```diff
@@ -57,6 +58,13 @@ message Policy {
   bool mitre_vectors_locked = 25;
   // Read-only field. Indicates the policy is a default policy if true and a custom policy if false.
   bool is_default = 26;
+
+  PolicySource source = 27;
+}
+
+enum PolicySource {
+  IMPERATIVE = 0;
+  DECLARATIVE = 1;
 }
```

Comment:
We had discussed allowing the value to be an open-ended string, so that end-users could set their own custom value, like "Github_action" or similar things. Did that idea get changed while I was out?

Reply:
While I was not part of that discussion, free form strings are a bad idea for validation et al.  There isn't a strong usecase to figure who owns the lifecycle of this policy, as much as the usecase to know the fact that the lifecycle of this policy is maintained by an entity outside of ACS. I will keep as is.